awx/api/views.py

```diff
@@ -2710,15 +2710,13 @@ def update_raw_data(self, data):
                 data['extra_vars'] = extra_vars
             ask_for_vars_dict = obj._ask_for_vars_dict()
             ask_for_vars_dict.pop('extra_vars')
-            if get_request_version(self.request) == 1:  # TODO: remove in 3.3
-                ask_for_vars_dict.pop('extra_credentials')
```

Comment:
No need to explicitly remove this key from the launch `defaults`, because it's no longer there.